packages/components/src/buttons/src/Button.module.css

```diff
@@ -103,11 +143,24 @@
     background-color: var(--background-color);
     color: var(--color);
     border: var(--border);
-    align-items:center
+    align-items:center;
+    text-decoration: none;
+}
+
+.hop-Button--icon-only {
+    column-gap: 0;
+}
+
+.hop-Button--icon-only.hop-Button--sm {
+    padding: var(--hop-Button-icon-only-sm-padding);
```

Comment:
padding does not exist in logical properties, yet, you could do padding-inline padding-block and use the splitted variables as noted above.